scripts/registry.sh

```diff
@@ -58,7 +58,7 @@ publish() {
     test -n "$prior" || prior=$(git rev-parse HEAD)
     git checkout -B lerna-publish
 
-    echo "$versions" | "$thisdir/set-versions.sh" "$d"
+    echo "$versions" | "$thisdir/set-versions.sh" .
```

Comment:
`$versions` may eventually blow out the max command length and could be captured in a temporary `git` blob object.